File: Lab programs/linearsearch.java
class linearsearch{
public static void main(String[] args){
int[] arr = {10,20,30,40,50,60,70};
int key = 70;		
for (int i =0; i<arr.length; i++)
{
if(arr[i]==key){
System.out.println("Key found at position: " + i);
}
}
}
}

File: Lab order/linear search.py
def linearsearch(arr,target):
    for i,value in enumerate(arr):
        if value==target:
            return i
        return-1
num=[6,7,8,9,4,3]
target=2
result=linearsearch(num,target)
if result !=-1:
    print("element are found")
else:
    print("element are not found")
import time
def display():
    print("find time complexity")
start=time.time()
display()
end=time.time()
print("the time complexity is",end-start)

